apps/comments-ui/src/actions.ts

```diff
@@ -408,6 +408,34 @@ async function openCommentForm({data: newForm, api, state}: {data: OpenCommentFo
     };
 }
 
+function highlightComment({
+    data: {commentId},
+    state,
+    setState
+}: {
+    data: { commentId: string | null };
+    state: EditableAppContext;
+    setState: (newState: Partial<EditableAppContext>) => void;
+}) {
+    const newState = {
+        ...state,
+        commentIdToHighlight: commentId
+    };
+    setState(newState);
```

Comment:
Can we just return this state so the action doesn't deviate too much from the other actions?